packages/core/data-transfer/src/strapi/providers/local-destination/index.ts

```diff
@@ -406,7 +370,7 @@ class LocalStrapiDestinationProvider implements IDestinationProvider {
               return callback();
             }
             const entry: IFile = await strapi.db.query('plugin::upload.file').findOne({
-              where: { id: fileEntitiesMapper[uploadData.id] },
+              where: { id: fileEntitiesMapper?.[uploadData.id] },
```

Comment:
What happens if it's indeed undefined? Should we throw an error? Allow continuing?

Reply:
Same question. Maybe we should have two checks, one before this query to ensure the id exists, and then after the query to ensure it actually returned the expected result. Does that make sense?